CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 2.x.0 (2020-08-DD)
```

Comment:
@corycaywood What would be the next version of our sdk?

Reply:
Probably the permissions feature will be in 2.2.0.

Reply:
I see, currently iOS also using [2020-08-dd](https://github.com/rakutentech/ios-miniapp/blob/master/CHANGELOG.md#2x0-2020-08-dd) as we haven't decided it finally.

Reply:
I think it's better to use something like `2.2.0 (in progress)` because that better communicates that it hasn't been released yet (also it doesn't commit to the release being in a certain month).

Reply:
Or maybe `2.x.x (in progress)` is even better?